modules/ROOT/pages/dataweave-language-introduction.adoc

```diff
@@ -301,33 +302,62 @@ The following table provides examples of invalid identifiers and a description o
 
 Valid identifiers cannot match any of the reserved DataWeave keywords:
 
-* `and`
-* `as`
-* `async`
-* `case`
-* `default`
-* `do`
-* `else`
-* `enum`
-* `false`
-* `for`
-* `fun`
-* `if`
-* `import`
-* `input`
-* `is`
-* `ns`
-* `null`
-* `or`
-* `output`
-* `private`
-* `throw`
-* `true`
-* `type`
-* `unless`
-* `using`
-* `var`
-* `yield`
+* `and`: +
+Reserved as a xref:dw-operators.adoc#logical_operators[logical operator].
+* `as`: +
+Reserved for xref:dataweave-types-coercion.adoc[type coercion].
+* `async`: +
+Reserved.
+* `case`: +
+Reserved for `case` statements, for example, to perform xref:dataweave-pattern-matching[pattern matching]
+or for use with the xref:dw-operators.adoc#update-operator[`update` operator].
+* `default`: +
+Reserved for assigning xref:dataweave-cookbook-defaults.adoc[default values] for parameters.
+* `do`: +
+Reserved for xref:dataweave-flow-control.adoc[`do`] statements.
+* `else`: +
+Reserved for use in if-else and else-if statements. See xref:dataweave-flow-control.adoc[Flow Control in DataWeave].
```

Comment:
HERE's the outlier:
```suggestion
Reserved for use in `if-else` and `else-if` statements +
See xref:dataweave-flow-control.adoc[Flow Control in DataWeave].
```

Reply:
DITTO globals throughout the rest. Copyedits to new stuff only.

Reply:
not `if-else` etc. for reasons described.